process-manager.js

```diff
@@ -126,7 +126,13 @@ class ProcessManager {
 	 * @return {ProcessWrapper}
 	 */
 	acquire() {
-		return this.processes.reduce((best, current) => current.load < best.load ? current : best); // Not sure if this is too obvious...?
+		let process = this.processes[0];
+		for (let i = 1; i < this.processes.length; i++) {
```

Comment:
This can be a for-of loop

Reply:
How should this be changed to a for-of loop? like this?

```js
let process;
for (const curProcess of this.processes) {
	if (!process || curProcess.load < process.load) {
		process = curProcess;
	}
}
return process;
```